pkg/user/admin/accept.go

```diff
@@ -9,17 +9,22 @@ import (
 func AcceptCmd() *cli.Command {
 	acceptCmd := &cli.Command{
 		Name:      "accept-admin",
-		Usage:     "user admin accept-admin <AccountAddress>",
+		Usage:     "user admin accept-admin --account-address <AccountAddress>",
 		UsageText: "Accepts a user to become admin who is currently pending admin acceptance.",
 		Description: `
 		Accepts a user to become admin who is currently pending admin acceptance.
 		`,
 		After: telemetry.AfterRunAction(),
-		Flags: []cli.Flag{
-			&flags.VerboseFlag,
-			&AccountAddressFlag,
-		},
+		Flags: acceptFlags(),
 	}
 
 	return acceptCmd
 }
+
+func acceptFlags() []cli.Flag {
+	cmdFlags := []cli.Flag{
+		&flags.VerboseFlag,
+		&AccountAddressFlag,
+	}
+	return append(cmdFlags, flags.GetSignerFlags()...)
```

Comment:
might want to sort it - https://github.com/Layr-Labs/eigenlayer-cli/blob/master/pkg/rewards/claim.go#L81 - so it's easier to read